match2/commons/match_group_display_matchlist.lua

```diff
@@ -206,15 +212,15 @@ Display component for a title in a matchlist.
 ]]
 function MatchlistDisplay.Title(props)
 	DisplayUtil.assertPropTypes(props, MatchlistDisplay.propTypes.Title)
+	local titleNode = html.create('div')
+		:css('width', props.width - 2 * 5 .. 'px')
```

Comment:
`2 * 5`? Why? More magic numbers

Reply:
Doesn't look like this has been addressed

Reply:
oops missed it